Sources/MapboxCoreNavigation/PassiveLocationDataSource.swift

```diff
@@ -235,15 +235,14 @@ extension TileEndpointConfiguration {
         guard let accessToken = credentials.accessToken, !accessToken.isEmpty else {
             preconditionFailure("No access token specified in Info.plist")
         }
-        let skuTokenProvider = SkuTokenProvider(with: credentials)
-        
+
         self.init(host: host,
                   dataset: "mapbox/driving",
                   version: tilesVersion,
                   token: accessToken,
                   userAgent: URLSession.userAgent,
                   navigatorVersion: "",
-                  skuTokenSource: skuTokenProvider,
+                  isFallback: false,
```

Comment:
@SiarheiFedartsou could you please elaborate on this field? How should we use it? Do we need to expose it?

Reply:
From the MapboxNavigationNative v49.0.1 internal release notes:

> `TileEndpointConfiguration` now allows passing the tile store instance directly. The `isFallback` should be specified when we choose a specific version as a result of the fallback observer notification.

It seems to have to do with `setFallbackVersionsObserver(_:)`, which we aren’t using here.

Reply:
@Udumft you're working with this part too, right?

Reply:
Right. `isFallback` will be covered by my PR